src/model.js

```diff
@@ -222,6 +222,16 @@ function queryArrayAt(i) {
   return r;
 }
 
+// Internal: Provides the implementation of the query array to empty and then refill its content and update its `meta`.
+function queryArrayReset(newInitialArray = [], meta) {
```

Comment:
Don't forget to document this method in the `buildQuery` docs.

Reply:
handled.
https://github.com/centro/transis/pull/53/files#diff-6eeed726084131b9a38134f8f89c2d4fL668